docs/API.md

```diff
@@ -1,47 +1,4 @@
 # Component API
 
 * [Component props](APIProps.md)
-* [Component methods](APIMethods.md)
-
-## Props quick reference
```

Comment:
Sure 👍🏽 However, aren't the definitions inferred by the tools using them?

PS. thanks @adidahiya @giladgray for the help! I never used TypeScript and I'm happy to learn something new!